Core/ComponentBaseClasses/elxTransformBase.h

```diff
@@ -293,7 +299,15 @@ class TransformBase :
   virtual void TransformPointsSomePointsVTK( const std::string filename ) const;
 
   /** Function to transform all coordinates from fixed to moving image. */
-  virtual void TransformPointsAllPoints( void ) const;
+  /** Legacy note: the function formerly called TransformPointsAllPoints was split into 
+      GenerateDeformationFieldImage for generating and WriteDeformationFieldImage for 
+      writing to disk*/
```

Comment:
Not sure we need to remind people of this. Sometimes it is just more confusing to keep legacy knowledge around.

Reply:
Ok, the comment could be a bit clearer. Maybe I should write the bigger plan here, which points out what and why things are legacy and is important to know for developers.
My plan is to split all Compute* functions (and others such as TransformPointsAllPoints) into Generate* and Write* functions, since that would facilitate a proper library interface. To keep everything functional during the transition period we need to keep the Compute* functions and make them just call Generate* and Write*, but mark them as legacy/deprecated.

Reply:
Sounds reasonable. Do you think we should merge this now as is or do you want to wait until the splitting of Compute* etc is done?

Reply:
I would like to go for a quick merge and only implement (and test) the things I need most right now. The rest will follow, hopefully. I do like to put the bigger plan as a comment in the code. For this I will make a new commit including the resolutions of other remarks.

Reply:
Great! I completely agree.